### PR TITLE
COS-2608: Switch from c9s content to RHEL 9.4 beta content

### PR DESCRIPTION
**Pull request by mike-nguyen**
RHEL 9.4 customer beta is now publicly available.

### CODE REVIEW

**Comment by mike-nguyen** [comment]
Need to check if the 9.4 beta content is mirrored in CI.

File: manifest-rhel-9.4.yaml [hunk 1]
```diff
@@ -113,22 +112,15 @@ postprocess:
...
+ # post-process script
+ - redhat-release
 
-# Packages pinned to specific repos in SCOS 9
+# Packages pinned to specific repos in RHCOS 9
 repo-packages:
-  # We always want the kernel from BaseOS
-  - repo: baseos
-    packages:
-      - kernel
```
Comment (travier) [inline manifest-rhel-9.4.yaml:127]: Any specific reason for removing this part?

Comment (travier) [reply manifest-rhel-9.4.yaml:127]: So this should be good.

**Comment by mike-nguyen** [comment]
Seeing 
```
BdsDxe: loading Boot0001 "UEFI Misc Device" from PciRoot(0x0)/Pci(0x2,0x0)
BdsDxe: failed to load Boot0001 "UEFI Misc Device" from PciRoot(0x0)/Pci(0x2,0x0): Access Denied
BdsDxe: No bootable option or device was found.
BdsDxe: Press any key to enter the Boot Manager Menu.
```
Which looks related to:
- https://github.com/openshift/os/issues/1237
- https://github.com/openshift/os/pull/1452

I tried building scos on master (not on this PR) and it had the same failure.

**Comment by sdodson** [comment]
As far as I know secureboot shim for 9.4 hasn't been signed yet. I'll see
if I can find a reference tomorrow but fine with snoozing the test for
another three weeks with goal of tracking this down by end of next sprint.

**Comment by mike-nguyen** [comment]
/override ci/prow/scos-9-build-test-qemu

The secure boot tests for c9s should be snoozed or skipped based on issues mentioned above.  Lets override the scos-9-qemu test and address it in a separate PR. This change does not touch anything scos related.

**Pull request merged**